app/puppet-code/print-config.go

```diff
@@ -0,0 +1,23 @@
+package app
+
+import (
+	"fmt"
+)
+
+type puppetConfig struct {
+	ServiceURL string `json:"service-url"`
+	Cacert     string `json:"cacert"`
+	TokenFile  string `json:"token-file"`
+	Token      string `json:"token"`
+}
+
+//GetConfig will return the configuration used
+func (puppetCode *PuppetCode) GetConfig() string {
```

Comment:
`app` is a wrapper for `api`, I think there is no need to implement this in `app`, could be directly implemented in CLI